cert_manager/Tiltfile

```diff
@@ -59,6 +59,7 @@ def deploy_cert_manager(registry="quay.io/jetstack", version="v1.1.0", load_to_k
 
     # 2. create a test certificate
     print("Testing cert-manager")
-    local("cat << EOF | kubectl apply -f - " + cert_manager_test_resources_versioned + "EOF", quiet=silent, echo_off=silent)
+    # The webhook may refuse connections initially (despite the deployment being Available), so try several times.
+    local("for i in 1 2 3 4 5 6; do (kubectl apply -f - <<EOF" + cert_manager_test_resources_versioned + "EOF\n) && break || sleep 15; done", quiet=silent, echo_off=silent)
```

Comment:
Maybe you can use the `range` syntax. But I am not so sure it works on Tilt!

Reply:
Looks like [range](https://github.com/bazelbuild/starlark/blob/master/spec.md#range) is a builtin function in the Starlark language. 

How would it help here though? I want to make a loop around the `local()` call but I can't catch the exception that happens when the `kubectl` command fails. I could put the output of `range(6)` into the command string but I'd have to strip off the square brackets.

It looks bad to hard-code `1 2 3 4 5 6` in here, but that seemed the most portable way. Better than relying on a utility like `seq`, for example.

Reply:
Ok fine!